core/src/main/java/jenkins/tasks/SimpleBuildWrapper.java

```diff
@@ -172,7 +299,7 @@ protected boolean runPreCheckout() {
         }
         @Override public boolean tearDown(AbstractBuild build, BuildListener listener) throws IOException, InterruptedException {
             if (c.disposer != null) {
-                c.disposer.tearDown(build, build.getWorkspace(), launcher, listener);
+                c.disposer.tearDown(build, build.getWorkspace(), this.launcher, listener);
```

Comment:
Note: not actually necessary. Not sure if there is some style guide saying to use `this.` when not necessary to disambiguate but merely as a reminder that the reference is to a field.

Reply:
I tend towards always qualifiying with `this` (or the declaring class for static items) because I find it really helps when reading new codebases. I can drop this diff, no problem.

Reply:
Whatever you find more legible is fine.

Reply:
Shrug. In this particular case, the class is so small it hardly matters either way, so I'll leave the change out of the PR.